### ISSUE TITLE
Failing to run the example

### ISSUE THREAD

**Issue by SamWyldKW** [trentrand/cypress-utils#10]
Hi, I've been trying to use this tool without much success.

I've cloned and installed cypress-utils and cloned cypress-example-kitchensink.

When trying to follow the example gif exactly I'm getting this error:
```
> cypress-utils stress-test location navigation

No Cypress spec files were found. You may have specified invalid file identifiers.

The following file identifiers were provided: "location", "navigation"

Your configuration specifies that test files are contained within the following directory:
`cypress/integration`

See help for the `--integrationFolder` command-line option if this is incorrect.
```

Any idea what I may be doing wrong?

**Comment by trentrand** [comment]
I'd guess it's because test files are located at directory [cypress/integration/examples](https://github.com/cypress-io/cypress-example-kitchensink/tree/master/) in the cypress-example-kitchensink repo.

Can you try passing `--integrationFolder ./cypress/integration/examples`?

**Comment by SamWyldKW** [comment]
#11 I've created this PR that fixes my use case. Without having any effect on the current functionality from what I understand. I found the issue was that I was missing `testFiles` and they were not being loaded because of this :D

**Comment by SamWyldKW** [comment]
All of my issues were fixed with the merge of #11 thanks @trentrand
